packages/node_modules/@webex/plugin-meetings/src/media/properties.js

```diff
@@ -22,12 +22,17 @@ export default class MediaProperties {
     this.shareTrack = options.shareTrack;
     this.remoteStream = options.remoteStream;
     this.remoteShare = options.remoteShare;
+    this.mediaSettings = {};
   }
 
   setMediaDirection(mediaDirection) {
     this.mediaDirection = mediaDirection;
   }
 
+  setSettings(type, values) {
```

Comment:
Would it be more consistent to do: 
```suggestion
  setMediaSettings(type, values) {
```
?